ios/GoodDollar/Info.plist

```diff
@@ -128,5 +134,17 @@
 	<false/>
 	<key>branch_app_domain</key>
 	<string>gooddollar.app.link</string>
```

Comment:
we need to make sure it opens wallet.gooddollar.org links

Reply:
I tested with wallet.gooddollar.org link, it works